themes/jaeger-docs/layouts/partials/home/articles.html

```diff
@@ -1,9 +1,46 @@
+{{- $maxBlogPosts := .Site.Params.maxBlogPosts }}
+{{- $json         := getJSON "https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/jaegertracing" }}
```

Comment:
is there a way for Hugo to pick the latest newsletter among all the blog posts? E.g. https://medium.com/jaegertracing/october-2018-newsletter-22c47541b8b1

Reply:
Yeah, you can just take the first item in the `$posts` array. What would you like to do with the most recent one?

Reply:
Okay, I added a `$mostRecent` variable that we can use in the future.

Reply:
it's not just the most recent, because some posts are not newsletters. We tag newsletters with the `newsletter` category on Medium. And we can use it in the main banner

![image](https://user-images.githubusercontent.com/3523016/48236139-e4e6eb00-e38e-11e8-9523-3304884cbc72.png)